### PR TITLE
Feat: Articles in Docs Sidebar

### PR DESCRIPTION
**Pull request by dennypradipta**
# Monika Pull Request (PR)  

## What feature/issue does this PR add  
1. This PR solves #535 

## How did you implement / how did you fix it  
1.  Use Medium RSS feeds
2. Create a simple API to fetch the RSS and process it
3. Display processed data to React

## How to test  
1. `npm run dev`
2. Go to <localhost:3000/articles>

## Screenshots
image

## Additional Context
I couldn't seem to get the images because the RSS feeds has no `image` or `media` field.

### CODE REVIEW

File: docs/src/manifests/manifest.json [hunk 1]
```diff
@@ -13,6 +13,11 @@
           "title": "Quick Start",
           "path": "/quick-start",
           "editUrl": "/quick-start.md"
+        },
+        {
+          "title": "Articles",
```
Comment (sapiderman) [inline docs/src/manifests/manifest.json:18]: "Articles" terlalu generic gk? Mungkin bisa lebih deskriptif? How-Tos? Self Help Corner? Knowledge base?

**Review by sapiderman — APPROVED** [review]
1 comment... Otherwise all good!

File: docs/src/manifests/manifest.json (hunk 1 above)
Comment (dennypradipta) [reply docs/src/manifests/manifest.json:18]: Need to confirm with @nicnocquee first :wink:

Comment (nicnocquee) [reply docs/src/manifests/manifest.json:18]: Self help corner sounds like monika is too depressing to use that we need to have a section called that. 😅 Knowledge base usually for FAQs I think. I think Articles is ok for now.

**Review by nicnocquee — APPROVED** [review]

**Pull request merged**